openlibrary/macros/AffiliateLinks.html

```diff
@@ -6,7 +6,7 @@
 
 $ alibris = "http://click.linksynergy.com/fs-bin/click?id=ECh45gHEoh4&offerid=99238.122856000&type=2&subid=0&tmpid=939&RD_PARM1=http%253A%252F%252Fwww.alibris.com/booksearch%253Fqsort%253Dp%2526isbn%253DXXX"
 
-$ amazon = "http://www.amazon.com/dp/XXX/?tag=internetarchi-20"
+$ amazon = "http://www.amazon.com/dp/XXX/?tag=" + affiliate_id('amazon')
```

Comment:
This seems like a big improvement -- we may want to utilize the same technique for our integration test(s)